NineChronicles.Headless.Executable/appsettings.json

```diff
@@ -110,5 +110,23 @@
         "NoCors": true,
         "Confirmations": 0,
         "ChainTipStaleBehaviorType": "reboot"
+    },
+    "IpRateLimiting": {
+        "EnableEndpointRateLimiting": true,
+        "StackBlockedRequests": true,
+        "RealIpHeader": "X-Real-IP",
+        "HttpStatusCode": 429,
+        "GeneralRules": [
+            {
+                "Endpoint": "*:/graphql",
+                "Period": "1s",
+                "Limit": 10
+            }
+        ],
+        "QuotaExceededResponse": {
+            "Content": "{{ \"message\": \"Whoa! Calm down, cowboy!\", \"details\": \"Quota exceeded. Maximum allowed: {0} per {1}. Please try again in {2} second(s).\" }}",
+            "ContentType": "application/json",
+            "StatusCode": 429
+        }
```

Comment:
[Just an opinion] In my understand, `IpRateLimiting` setting is one of headless's options, right? So I suggest moving this settings into `Headless` section.
Welcome any ideas :raised_hands:

Reply:
Thanks for you feedback!

The `IPRateLimiting` setting in `appsettings.json` is used by the [AspNetCoreRateLimit](https://github.com/stefanprodan/AspNetCoreRateLimit) package. The setting is retreived here (https://github.com/area363/NineChronicles.Standalone/blob/feat/rate-limit/NineChronicles.Headless.Executable/Program.cs#L314) and passed down to `GraphQLService.cs`.

So, the `IPRateLimiting` setting isn't necessarily a headless option. However, because there should be a way to enable or disable `IPRateLimiting`, I've added the `--rate-limit` to the headless option which is `null` in default (meaning rate limiting is disable).

Let me add `RateLimit` inside the appsettings headless option and request another feedback from you. 🙏